Sprite stuff: missing belt and suit storage sprites among others

necromanceranne opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Adds a bunch of sprites including:

A basic nullrod sprite
~~Belt sprites for the claymores~~ lol nevermind
Backsprites for the cult blade and changed the chaplain's darkblade to fit on the back.
Suit storage sprites for the dark scythe, the cultblade, the spellblade, the claymores, katana and the nullrod

Suit storage sprites for a whole host of high powered firearms. You can visibly tell someone has shotguns or sniper rifles in their armor vest. How neat.

You can't conceal stunprods in armor suit storage anymore.

Nullrod cultsword (darkblade) also uses a uniquer inhand. Ye.

Also knives and a rolling pin belt sprite, hecc!

I also did some other stuff look there are a lot of sprites and I want to sleep please leave me alone nullrods are fucking stupid.

## Why It's Good For The Game

Pink begone!

## Changelog
:cl:
tweak: Makes the darkblade fit on your back instead of your belt.
imageadd: Adds a whole bunch of missing sprites for: the nullrod, cult blade, spellblade, claymores and dark scythe, among others.
imageadd: Updates the on-mobs for shotguns a little, and fixes several high powered firearms being concealable in suit storage slots.
imageadd: Makes stunprods visible in suit storage slots.
imageadd: Knife sprites for you belt and suit storage!
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

necromanceranne committed:
Does a whole bunch of sprites for belt, belt_mirror and so on, and makes the darkblade use a unique version of the longsword sprite and equippable in the back slot.

necromanceranne commented:
That's weird. Hold on a tick.

necromanceranne committed:
Merge branch 'master' into whole_bunch_of_belt_sprites

ArcaneMusic commented:
The real criminals where whoever made half the nullrods parent to claymores for christs sake.

necromanceranne committed:
ah someone already got to some of these sprites and I was a few commits behind, nice

necromanceranne commented:
The nullrods are a fucking mess, my dude.

necromanceranne committed:
Forgot one

necromanceranne commented:
What the fuck.

necromanceranne committed:
Please leave me alone travis this is a sprite that exists why are you angry?

Rohesie commented:
Code is fine, pending art review.

Rohesie commented on code/modules/modular_computers/hardware/portable_disk.dm:
```diff
@@ -2,7 +2,7 @@
 	desc = "Removable disk used to store data."
 	power_usage = 10
-	icon_state = "datadisk6"
+	icon_state = "datadisk0"
```
Is this an intended change? Looks undocumented, or am I missing something?

necromanceranne replied:
It..I don't really know why, but travis was failing on this for some reason since apparently it wouldn't recognize this particular sprite, despite it being in the dmi it's trying to use. I probably should have asked about it.

Rohesie replied:
that's a bug with the automated test, feel free to ignore it, it comes and goes and won't be fixed by this

necromanceranne replied:
fair enough. I'll just kick this back to what it was. (not that it really matters)

necromanceranne committed:
Fixed pulse destroyers using the wrong sprite, made suit storage sprites for a whole lot more weapons that probably shouldn't be concealed by suit storage

necromanceranne commented:
I added in a fuckton of suit storage sprites for various guns that absolutely shouldn't ever ben concealable in suit storage. They weren't previously due to becoming the weird pink sprites, but that's really no better.

I also changed the slings on the various shotguns, mostly because they looked really depressing. This one might be quite subtle, but it's at least not a grey smear across a person's chest.

necromanceranne committed:
Minor fix to the esniper sprites

necromanceranne committed:
maybe travis won't hate me anymore

Twaticus reviewed (approved):
LETS FUCKING GOOOOO

nice job :+1:

necromanceranne committed:
Even more sprite fixes and additions. Kitchen implements.

necromanceranne commented:
Sorry to give even more work but I recently played chef and saw a bunch of pink. So I've added some knives and a rolling pin to the list of new sprites.

Do note, the real cult dagger doesn't have the new cult knife sprite, only the wizard one on their ship. I'm not touching on cult balance with this, but the sprite exists if someone wants to hook that up.

Rohesie commented:
Code still fine, new art review pending.
